eth_worker/eth_manager/task_interfaces/composite.py

```diff
@@ -4,33 +4,33 @@
 import datetime
 
 import config
-from eth_manager import persistence_interface, utils, w3, red
+from eth_manager import persistence_module, utils, w3, red
 from eth_manager.task_interfaces.regular import (
     deploy_contract_task,
     transaction_task,
     send_eth_task,
     synchronous_call,
-    await_task_success,
+    await_blockchain_success_evil,
```

Comment:
What's up with the evil twin here?

Reply:
hahahaha oh that was from what this pr was orginally supposed to be, which was remove any celery tasks that may cause worker lock and crash the entire thing. Named it evil so i could gradually refactor it out. I'm gonna leave this cos its fun lol!